apps/etterlatte-saksbehandling-ui/client/src/components/person/uhaandtereHendelser/HistoriskeHendelser.tsx

```diff
@@ -60,6 +70,55 @@ const HistoriskeHendelser = (props: Props) => {
   )
 }
 
+const InstitusjonsoppholdBegrunnelse = ({ id }: { id: string }) => {
+  const [begrunnelse, hentInstbegrunnelseData] = useApiCall(hentInstitusjonsoppholdData)
+
+  useEffect(() => {
+    hentInstbegrunnelseData(id)
+  }, [])
+
+  return (
+    <>
+      {isPending(begrunnelse) && <Spinner visible={true} label="Henter vurdering" />}
+      {isFailure(begrunnelse) && <p>Fant ingen begrunnelse for hendelsen</p>}
+      {isSuccess(begrunnelse) && (
+        <div>
+          <Heading spacing size="small" level="3">
+            Institusjonsoppholdshendelsen er vurdert
+            <SuccessColored aria-hidden={'true'} />
+          </Heading>
+          <p>
+            Er dette en institusjon som kan gi reduksjon av ytelsen? - <b>{begrunnelse.data.kanGiReduksjonAvYtelse}</b>
+          </p>
+          <p>Kommentar - {begrunnelse.data.kanGiReduksjonAvYtelseBegrunnelse}</p>
+          <p>
+            Er oppholdet forventet å vare lenger enn innleggelsesmåned + tre måneder? -
+            <b>{begrunnelse.data.forventetVarighetMerEnn3Maaneder}</b>
+          </p>
+          <p>Kommentar - {begrunnelse.data.forventetVarighetMerEnn3MaanederBegrunnelse}</p>
+          <VilkaarVurdertInformasjon>
+            <Detail>Manuelt av {begrunnelse.data.saksbehandler.ident}</Detail>
```

Comment:
Er dette mest for sporbarheit, eller er det noko saksbehandlarane er forventa å bry seg om? I så fall ville eg heller hatt namn enn ident.

Reply:
Det er et godt innspill, det er gjort på lik linje som vurderingene man finner på Søknadsoversikten. Der holder det tydeligvis med ident. Skal man ha navn må vi gjøre et oppslag mot axsys i tillegg om jeg husker riktig.